acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/factory/PantheonNodeFactory.java

```diff
@@ -120,6 +121,17 @@ public PantheonNode createArchiveNodeWithP2pDisabled(final String name) throws I
             .build());
   }
 
+  public PantheonNode createArchiveNodeWithP2pDisabledAndAdmin(final String name)
+      throws IOException {
+    return create(
+        new PantheonFactoryConfigurationBuilder()
+            .setName(name)
+            .setP2pEnabled(false)
+            .setJsonRpcConfiguration(jsonRpcConfigWithPermissioningAndAdmin())
+            .webSocketEnabled()
```

Comment:
Why do we need to enable websockets for this node?

Reply:
👍